frontend/packages/dev-console/integration-tests/features/operators.feature

```diff
@@ -126,3 +126,24 @@ Scenario: Install Web Terminal operator from Operator Hub page
    And user installs the Web Terminal operator with default values
    Then user will see a modal with title "Web Terminal"
    And user will see a View Operator button
+
+
+@regression, @smoke
+Scenario: Install Red Hat Integration - Camel Operator
+   Given user has installed OpenShift Serverless Operator
+   And user is at Operator Hub page with the header name "OperatorHub"
+   When user searches and installs the Red Hat Integration - Camel Operator with default values
+   Then user will see a modal with title "knative Apache Camel Operator"
```

Comment:
should this be `Red Hat Integration - Camel K`?

Reply:
Yeah!!!. Updated

Reply:
`Then user will see a modal with title "knative Apache Camel Operator"` i don't see this updated